docs/ops/indexing.rst

```diff
@@ -68,17 +57,79 @@ what type of **units** the measurement is in, a **nodata** value, and even a way
 of specifying **bit level descriptions** or the **spectral response** in the
 case of reflectance data.
 
-A set of example Product definitions are supplied in
-`docs/config_samples/dataset_types` to cover some common Geoscience Australia
-and other Earth Observation Data.
+A product definition example:
```

Comment:
I did wonder if an even simpler version could be done, but then thought this one has the merit of being both real and very simple even if it is a bit long if you only want one band.